### ISSUE TITLE
False positive for default value

### ISSUE THREAD

**Issue by Goldziher** [snok/flake8-type-checking#120]
I have this:

```python
# starlite/exceptions/utils.py
from typing import Any, Dict, List, Optional, Union

from pydantic import BaseModel
from starlette.exceptions import HTTPException as StarletteHTTPException
from starlette.status import HTTP_500_INTERNAL_SERVER_ERROR

from starlite.enums import MediaType
from starlite.response import Response

from .exceptions import HTTPException

__all__ = ["create_exception_response"]


class ExceptionResponseContent(BaseModel):
    detail: Optional[str]
    extra: Optional[Union[Dict[str, Any], List[Any]]]
    status_code: int = HTTP_500_INTERNAL_SERVER_ERROR


def create_exception_response(exc: Exception) -> Response:
    """
    Constructs a response from an exception.

    For instances of either `starlite.exceptions.HTTPException` or `starlette.exceptions.HTTPException` the response
    status code is drawn from the exception, otherwise response status is `HTTP_500_INTERNAL_SERVER_ERROR`.

    Args:
        exc (Exception): Any exception.

    Returns:
        Response
    """
    if isinstance(exc, (HTTPException, StarletteHTTPException)):
        content = ExceptionResponseContent(detail=exc.detail, status_code=exc.status_code)
        if isinstance(exc, HTTPException):
            content.extra = exc.extra
    else:
        content = ExceptionResponseContent(detail=repr(exc))
    return Response(
        media_type=MediaType.JSON,
        content=content.dict(exclude_none=True),
        status_code=content.status_code,
    )
```

And this error is found:

```shell
starlite/exceptions/utils.py:5:1: TC002 Move third-party import 'starlette.status.HTTP_500_INTERNAL_SERVER_ERROR' into a type-checking block
```

It apparently doesn't identify the default value as a valid use.

**Comment by sondrelg** [comment]
Thanks @Goldziher, that's weird!

Replicates for this as well
```python
from logging import INFO

from starlette.status import HTTP_500_INTERNAL_SERVER_ERROR

class C:
    level: int = INFO
    status: int = HTTP_500_INTERNAL_SERVER_ERROR
```

I'll take a look when I get a chance

**Comment by sondrelg** [comment]
Fix out now in v2.1.1 🙂
